client/src/pages/Feed.js

```diff
@@ -258,6 +260,41 @@ const Feed = () => {
     dispatchAction(TOGGLE_STATE, "showFilters");
   }
 
+  const handlePostLike = (postId) => {
+    const { history, isAuthenticated, user } = props;
+
+    /* added here because userId not working */
+    sessionStorage.removeItem("likePost");
+
+    if (isAuthenticated) {
+      const endPoint = `/api/posts/${postId}/likes/${user && user.userId}`;
+      const post = posts.posts.filter((post) => post._id === postId);
+
+      if (user) {
```

Comment:
Why keep this as a separate check from line 269? On 269 why not ` if (isAuthenticated && user)` ?

Reply:
The current user object is not been created after login, if I do (isAuthenticated && user) it will always throw false and creates an infinite loop.

Reply:
And I asked about how to obtain userId information and I didn't receive any answer about this field.